### ISSUE TITLE
Need rename GitHub repo, how correct update spec? 

### ISSUE THREAD

**Issue by ivanvorobei** [CocoaPods/CocoaPods#9315]
I have pod, now it installing as:

```
pod 'OldName'
```

And all spec links using `/OldName/`. Need rename repo on GitHub and update pod name to `NewName`. How to do this to save current users who will use the old name?

**Comment by dnkoutso** [comment]
You have to move the github repo and then update your podspec `s.source` DSL to point to the new repo. Not sure if GH handles re-directs.

For now closing and we can continue discussion here.

**Comment by ivanvorobei** [comment]
I did not want to transfer to a new repository. I will describe the steps that I did for the desired result:

1. Push new Podspec with new links to GitHub and extra line
```
spec.deprecated_in_favor_of = 'NewNamePod'
```
2. Fast Rename Github repo to `NewName`. Github using redirect and links to `/OldName/` redirect to new name. 

3. In renamed repo create new Podspec version, remove `deprecated_in_favor_of`. Also rename pod name, Podspec name and all other. Push Podspec with new name. 

As result when user install pod `OldName`, it show warning what need use `NewName`, and installing last version for `OldName`. 

Soft migration. Sorry for my English. 
Maybe somebody need it.
